dev/io.openliberty.transport.http_fat/publish/files/AccessListsTests_testA10_server.xml

```diff
@@ -0,0 +1,24 @@
+<!-- Copyright (c) 2017 IBM Corporation and others. All rights reserved. 
```

Comment:
can you fix the copyright date, and remove the `CLIENT_HOST CLIENT_ADDR CLIENT_PORT` from these files (unless I'm missing something and the string is used somehow)?

Reply:
Ok will do.

Reply:
Done.